### PR TITLE
Use GitHub Actions

### PR DESCRIPTION
**Pull request by andrewrabert**

### CODE REVIEW

File: .github/workflows/deploy.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    branches-ignore: ["*"]
+    tags: ["*"]
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    if: github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags') && github.repository == 'sprockets/sprockets.http'
+    container: python:3.9-alpine
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v1
```
Comment (aremm) [inline .github/workflows/deploy.yaml:13]: Probably should use [actions/checkout@v2](https://github.com/actions/checkout)

File: .github/workflows/deploy.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+      - name: Build package
+        run: python3 setup.py sdist
```
Comment (aremm) [inline .github/workflows/deploy.yaml]: Just to note, it was building wheel prior

File: .github/workflows/deploy.yaml (hunk 1 above)
Comment (andrewrabert) [reply .github/workflows/deploy.yaml:13]: Done

File: .github/workflows/deploy.yaml (hunk 2 above)
Comment (andrewrabert) [reply .github/workflows/deploy.yaml]: Added wheel building

**Pull request merged**